images/installer/Dockerfile.upi.ci

```diff
@@ -20,6 +20,8 @@ COPY --from=builder /go/src/github.com/openshift/installer/upi /var/lib/openshif
 
 COPY --from=govc /govc /bin/govc
 
+RUN curl https://packages.microsoft.com/config/rhel/8/prod.repo | tee /etc/yum.repos.d/microsoft.repo
```

Comment:
```suggestion
RUN curl https://packages.microsoft.com/config/rhel/8/prod.repo -o /etc/yum.repos.d/microsoft.repo
```

Reply:
I'd prefer if we steered away from `curl`ing deps all the time and instead used an existing container image, the same way we get `govc`, for example.

Reply:
I'll try to clean that up.  Thanks for the feedback.